From: John Ronciak <john.ronciak@intel.com>
Subject: add missing ethtool hooks
Acked-by: Karsten Keil <kkeil@novell.com>
Reference: bnc#435551


Patch to fix the missing ethtool_ops_[sg]et function pointers

This patch needs to be applied to the igb driver to add the function pointers
to the ethtool_ops_[gs]et funcitons. 

---
 drivers/net/igb/igb_ethtool.c |    4 ++++
 1 file changed, 4 insertions(+)

--- a/drivers/net/igb/igb_ethtool.c
+++ b/drivers/net/igb/igb_ethtool.c
@@ -2029,6 +2029,10 @@ static struct ethtool_ops igb_ethtool_op
 	.get_ethtool_stats      = igb_get_ethtool_stats,
 	.get_coalesce           = igb_get_coalesce,
 	.set_coalesce           = igb_set_coalesce,
+#ifdef NETIF_F_LRO
+        .get_flags              = ethtool_op_get_flags,
+        .set_flags              = ethtool_op_set_flags,
+#endif
 };
 
 void igb_set_ethtool_ops(struct net_device *netdev)
